src/test/kotlin/net/maizegenetics/phgv2/cli/AlignAssembliesTest.kt

```diff
@@ -170,6 +170,31 @@ class AlignAssembliesTest {
 
     }
 
+    @Test
+    fun testAssemblyRelativePathDir() {
+        // This function tests that ggsave can write when the assembly path is relative
+        // to the output directory.  This is really a test of the ggsave function, not the
+        // AlignAssemblies function, but it verifies a fix added to AlignAssemblies.
+
+        // create a directory name userPlotDir under the user's current working directory
+        val userPlotDir = "userPlotDir"
+        val userPlotDirPath = "${System.getProperty("user.dir")}/$userPlotDir"
+        File(userPlotDirPath).mkdir()
+
+        val outputDir = "lynnOutput/alignment_files"
```

Comment:
Should this be something other than lynnOutput?

Reply:
actually, that line isn't used and should be deleted - is leftover from before cleanup.  See above, we're actually using "userPlotDir"

I did fix the code to use the parameter "userPlotDir" instead of the hard-coded "userPlotDir" in the test call.